tests/ios/frameworks/sources-with-prebuilt-binaries/.ruby-version

```diff
@@ -0,0 +1 @@
+2.7.4
```

Comment:
Totally open to changing this to whatever folks want. But it was helpful in my case to fix the following error: 
```
╰─ bundle install         
Warning: the running version of Bundler (2.1.4) is older than the version that created the lockfile (2.3.7). We suggest you to upgrade to the version that created the lockfile by running `gem install bundler:2.3.7`.
Fetching gem metadata from https://rubygems.org/.......
activesupport-7.0.4.3 requires ruby version >= 2.7.0, which is incompatible with the current version, ruby 2.6.10p210
```

Reply:
I'd probably optimize this for whatever the Github runners have on them for now: https://github.com/actions/runner-images/blob/main/images/macos/macos-12-Readme.md#ruby

So perhaps `2.7.8`?
```suggestion
2.7.8
```

Reply:
`macos-11` also have `2.7.8` so I'd say we go with that: https://github.com/actions/runner-images/blob/main/images/macos/macos-11-Readme.md#ruby